content/docs/sidebar.json

```diff
@@ -191,6 +191,37 @@
         "label": "doctor",
         "slug": "doctor"
       },
+      {
+        "label": "experiments",
+        "slug": "exp",
+        "source": "exp/index.md",
+        "children": [
+          {
+            "label": "exp checkout",
+            "slug": "checkout"
+          },
+          {
+            "label": "exp diff",
+            "slug": "diff"
```

Comment:
Should be ordered by importance/ expected usage frequency. Perhaps as listed on the top of [the index page](https://dvc-landing-experiments-ixzhup.herokuapp.com/doc/command-reference/exp)? "show, checkout, diff, run, resume, and gc."

Reply:
I imagine run is more important or "comes before" diff, though. So let's just come up with a logical order and reflect it anywhere they're listed please. Thanks

Reply:
I think we use alphabetic order everywhere in the command reference?

Reply:
Top of the index page and sidebar are alphabetical for now.

Reply:
> I think we use alphabetic order everywhere in the command reference?

Some subcommands (e.g. metrics, plots) are in a "logical" order. `remote` subcommands are abc indeed... So it's not consistent unfortunately. I guess we can address that later separately, resolving.